.github/workflows/code.yml

```diff
@@ -98,7 +98,7 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.8"
+          # - "3.8"  # TODO: Breaking
```

Comment:
We're really close to EOL for Py3.8.  This would push the entire repo to drop it.  What breaks for Py3.8 now?

Reply:
I came across this [import error](https://github.com/gfabbris/apstools/actions/runs/10044498274/job/27760058119#step:15:94).